modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js

```diff
@@ -241,6 +241,8 @@ AUI.add(
 					_changeFieldTypeMenu: function(fieldType) {
 						var instance = this;
 
+						instance._disableHeader();
```

Comment:
Hey @alinedoleron what do you think we rename _disableHeader for _disableFieldTypeMenu or _disableSidebarHeader or _disableSidebarMenu, as result of the menus from the sidebar will be disabled, right?